deploy/nv-ipam.yaml

```diff
@@ -52,15 +52,14 @@ spec:
         app: nv-ipam-node
         name: nv-ipam-node
     spec:
-      hostNetwork: true
```

Comment:
@moshe010 , it will now depend on CNI, do we need to introduce this dependency ?

Reply:
I think you are right if we want to support primary network it has to be hostNetwork: true and also the controller.